Description: Allow package to build with glibc-2.16
 'gets' has been removed from glibc-2.16 but gnulib still refers to it
 causing build failures.
 The simple fix for now is just to put a check around the reference in
 lib/stdio.in.h
 A new release made with a new gnulib is a better fix.
 .
 tar (1.26-5) unstable; urgency=low
 .
   * Fix gnu/stdio.in.h to build with glibc-2.16 (gets removed)
Author: Wookey <wookey@wookware.org>
Last-Update: 2012-11-5
Bug : http://lists.gnu.org/archive/html/bug-gnulib/2012-03/msg00186.html
Bug-Debian: http://bugs.debian.org/687986
Forwarded: not-needed

--- tar-1.26.orig/gnu/stdio.in.h
+++ tar-1.26/gnu/stdio.in.h
@@ -164,7 +164,9 @@ _GL_WARN_ON_USE (fflush, "fflush is not
    so any use of gets warrants an unconditional warning.  Assume it is
    always declared, since it is required by C89.  */
 #undef gets
+#ifdef HAVE_RAW_DECL_GETS
 _GL_WARN_ON_USE (gets, "gets is a security hole - use fgets instead");
+#endif
 
 #if @GNULIB_FOPEN@
 # if @REPLACE_FOPEN@
